views/create.ejs

```diff
@@ -0,0 +1,131 @@
+          </select>
+          <h3>Tags *</h3>
+          <input type="text" id="tags" placeholder="List of tags eligible">
+          <h3 class="whitelist">Whitelist</h3>
+          <input type="text" id="whitelist" class="whitelist" placeholder="Whitelisted users">
+          <h3 class="blacklist">Blacklist</h3>
+          <input type="text" class="blacklist" id="blacklist" placeholder="Blacklisted users">
+          <h3>Vote at Voting Power percentage *</h3>
+          <input type="number" id="vote_when" placeholder="100"/>
+          <h3>Maximum number of daily upvotes per user *</h3>
+          <input type="number" id="max_upvote" placeholder="1"/>
+          <button id="create_community">Create!</button>
+        </div>
+      </div>
+
+
+			<footer id="footer" class="align-center">
+				<p>Created with <span class="heart">♥</span> for Steem and Utopian</p>
+				<p>Idea by <a href="https://steemit.com/@flauwy" target="_blank">@flauwy</a>, development by <a href="https://steemit.com/@stoodkev" target="_blank">@stoodkev</a>, design by <a href="https://steemit.com/@andrejcibik" target="_blank">@andrejcibik</a>.</p>
+      </footer>
+
+    </div>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+    <script src="https://cdn.steemjs.com/lib/latest/steem.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/parse/1.11.0/parse.min.js"></script>
+    <script>
+
+
+	/* Mobile menu openning */
+	$( document ).ready(function() {
+
+    //Show whitelist or blacklist according to the type of community
+    $("#type_community").change(function(){
+        $(".whitelist").toggle($("#type_community option[name='whitelist']").prop("selected")==true);
+        $(".blacklist").toggle($("#type_community option[name='open']").prop("selected")==true);
+
+    });
+
+    //If all fields are completed, send back the information to create the community
+    $("#create_community").click(function(){
+      if($("#name").val()!=""
+        &&$("#description").val()!=""
+        &&$("#image").val()!=""
+        &&$("#tags").val()!=""
+        &&$("#max_upvote").val()!=""
+        &&$("#vote_when").val()!="")
+        {
+          var obj={
+            name:$("#name").val(),
+            description:$("#description").val(),
+            image:$("#image").val(),
+            tags:$("#tags").val(),
+            max_upvote:$("#max_upvote").val(),
+            vote_when:$("#vote_when").val(),
+            type_community:$("#type_community option:selected").val(),
+            administrators:$("#admins").val(),
+            moderators:$("#moderators").val(),
+            whitelist:$("#whitelist").val(),
+            blacklist:$("#blacklist").val(),
+            owner:"<%=account.name %>"
+          };
+          window.location.href="/createCommunity?community="+JSON.stringify(obj);
+        }
+    });
+
+		var menuMainButton = $('.menu-main-button');
+		menuMainButton.click(function(){

```

Comment:
instead of checking for the class here, just use [`.toggleClass('open-menu')`](http://api.jquery.com/toggleclass/)